### ISSUE TITLE
Feature request: screensaver_start_navigation_path

### ISSUE THREAD

**Issue by kwithus** [j-a-n/lovelace-wallpanel#159]
Would be nice to have this option as well, I know it seems useless, as you don’t see the tab, once the screensaver has started. However, This would make sure that it’s always the default view behind the screensaver and when I stop it manually, the correct tab is displayed. Yes, there is the screensaver_stop_navigation_path feature, however, this interferes with the stop_screensaver_on_location_change feature, which I use to stop the screensaver when I move to a specific tab by clicking on a button within the screensaver.

**Comment by j-a-n** [comment]
Even though it says `screensaver_stop_navigation_path`, this location is set right after the screensaver starts.
How does `stop_screensaver_on_location_change` interfere with this function?

**Comment by kwithus** [comment]
Well, I just added the screensaver-stop-navigation-path again and now it works as expected. Now  idea why it did not work before.
The problem I had was that when I had this and the stop-screensaver-on location-change, both implemented, the effect was always, that when I changed the location with pressing a button, the screensaver stopped and instead of switching to the selected tab, it showed the tab of the screensaver -stop-navigation-path.
Well, now it works as expected
